src/keybindings.c

```diff
@@ -2411,10 +2410,10 @@ static gboolean cb_func_select_action(guint key_id)
 			editor_select_paragraph(doc->editor);
 			break;
 		case GEANY_KEYS_SELECT_WORDPARTLEFT:
-			sci_send_command(sci, SCI_WORDPARTLEFTEXTEND);
+			sci_send_command(doc->editor->sci, SCI_WORDPARTLEFTEXTEND);
 			break;
 		case GEANY_KEYS_SELECT_WORDPARTRIGHT:
-			sci_send_command(sci, SCI_WORDPARTRIGHTEXTEND);
+			sci_send_command(doc->editor->sci, SCI_WORDPARTRIGHTEXTEND);
 			break;
```

Comment:
those two (`WORDPARTLEFT` and `WORDPARTRIGHT`) could also work with the `focusw`